maud/tests/html_test.rs

```diff
@@ -0,0 +1,36 @@
+#[cfg(test)]
+#[macro_export]
+macro_rules! html_test {
```

Comment:
This code generation is pretty clever, but I think from a code coverage point of view we can get away with just a few specific tests for `html_to!`.

Reply:
Sure, just keeping this for now until you're ok with the rest